include/deal.II/lac/la_vector.templates.h

```diff
@@ -26,6 +26,51 @@ DEAL_II_NAMESPACE_OPEN
 namespace LinearAlgebra
 {
   template <typename Number>
+  void Vector<Number>::reinit(const size_type size,
+                              const bool omit_zeroing_entries)
+  {
+    ReadWriteVector<Number>::reinit(size, omit_zeroing_entries);
```

Comment:
Is there any reason why you redefine the `reinit` method in `Vector` when it already exists (as a virtual function) in `ReadWriteVector` and does just call the base class?